modelci/hub/deployer/pytorch/docker-env.env

```diff
@@ -0,0 +1,2 @@
+CUDA_DEVICE_ORDER=PCI_BUS_ID
```

Comment:
do not force to add `docker-env.env`, it is ignored in `modelci/hub/deployer/.gitigrnoe`